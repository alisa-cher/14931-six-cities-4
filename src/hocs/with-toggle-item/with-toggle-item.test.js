import React from 'react';
import renderer from 'react-test-renderer';
import withToggleItem from "./with-toggle-item";

it(`Should render WithToggleItem component`, () => {
  const MockComponent = () => <div />;
  const NewComponent = withToggleItem(MockComponent);

  const tree = renderer
    .create(
        <NewComponent/>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
