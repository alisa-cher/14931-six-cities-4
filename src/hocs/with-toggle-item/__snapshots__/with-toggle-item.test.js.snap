// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render WithToggleItem component 1`] = `<div />`;
